src/stream/src/executor/source/list_executor.rs

```diff
@@ -99,6 +99,10 @@ impl<S: StateStore> FsListExecutor<S> {
                 .collect::<Vec<_>>();
 
             let res: Vec<(Op, OwnedRow)> = rows.into_iter().flatten().collect();
+            if res.is_empty(){
+                return Err(StreamExecutorError::connector_error(
+                    "The file in source cannot be listed. Please check the log to confirm whether the source configuration is correct."));
```

Comment:
why is empty res translated into source cannot be listed? isn't it possible that the source can be listed but there are no files in it?